modules/thermal_hydraulics/doc/content/modules/thermal_hydraulics/source/components/VolumeJunction1Phase.md

```diff
@@ -1,28 +1,49 @@
 # VolumeJunction1Phase
 
-!syntax description /Components/VolumeJunction1Phase
+This is a [flow junction](component_groups/flow_junction.md) that has a volume
+and can connect 2 or more [FlowChannel1Phase.md] components in any orientation.
 
-This component implements a junction model for 1-phase flow that has a volume.
-A form loss coefficient
-[!param](/Components/VolumeJunction1Phase/K) can be supplied by the user. The source term on the momentum equation is:
+## Formulation
 
-!equation id=momentum_source
-S^{\text{momentum}} = - K (p_0 - p) A  \hat{n}_1,
+See [!cite](relap7theory) for a description of this junction formulation.
 
+### Form Losses
+
+Complex multidimensional interactions inside the junction cannot be practically
+modeled mechanistically but are instead approximated using a form loss factor
+$K$, which gives rise to source terms on the momentum and energy equations:
+
+!equation id=momentum
+\mathbf{S}^{\text{momentum}} = - K (p_{0,1} - p_1) A_\text{ref} \frac{\mathbf{u}_1}{\|\mathbf{u}_1\|} \eqc
+
+!equation id=energy
+S^{\text{energy}} = - K (p_{0,1} - p_1) A_\text{ref} \|\mathbf{u}_1\| \eqc
 
 where
 
-- $p_0$ is the stagnation pressure of the first [flow channel](FlowChannel1Phase.md) connected to the junction in [!param](/Components/VolumeJunction1Phase/connections),
-- $p$ is the pressure of the first [flow channel](FlowChannel1Phase.md) connected to the junction,
-- A is the flow area of the first [flow channel](FlowChannel1Phase.md) connected to the junction or [!param](/Components/VolumeJunction1Phase/A_ref) if it was supplied by the user, and
-- $\hat{n}_1$ is the direction of the flow in the first [flow channel](FlowChannel1Phase.md) connected to the junction.
+- $p_{0,1}$ is the stagnation pressure of the first flow channel (see [#usage]),
+- $p_1$ is the static pressure of the first flow channel,
+- $A_\text{ref}$ is the reference cross-sectional area, and
+- $\mathbf{u}_1$ is the velocity in the first connected flow channel.
+
+## Usage id=usage
```

Comment:
It's because I was second-guessing our decision to put Usage before Formulation - in the Usage section, I'm always referring to symbols defined in the formulation section.

I will move this for now to be consistent - we can change them all later at once if we want.